ParallelMultiply/ParallelMultiply/FileMatrix.cs

```diff
@@ -0,0 +1,125 @@
+// <copyright file="FileMatrix.cs" company="NematMusaev">
+// Copyright (c) PlaceholderCompany. All rights reserved.
+// </copyright>
+namespace ParallelMultiplication;
+
+using System.Text;
+using System.Threading;
+
+/// <summary>
+/// Creates and generate matrixes, gets file with matrix.
+/// </summary>
+public class FileMatrix
+{
+    /// <summary>
+    /// Write matrix to file.
+    /// </summary>
+    /// <param name="matrix"> Matrix wich will be written to file. </param>
+    /// <param name="path"> Path with matrix. </param>
+    public static void WriteToFile(int[,] matrix, string path)
+    {
+        var stringList = new List<string> { };
+        var line = new StringBuilder();
+
+        for (var i = 0; i < matrix.GetLength(0); ++i)
+        {
+            for (var j = 0; j < matrix.GetLength(1); ++j)
+            {
+                line.Append($"{matrix[i, j]} ");
+            }
+
+            stringList.Add(line.ToString()[..^1]);
+            line = new StringBuilder();
+        }
+
+        File.WriteAllLines(path, stringList);
+    }
+
+    /// <summary>
+    /// Generate matrix.
+    /// </summary>
+    /// <param name="sizeRows">Size of matrix at rows. </param>
+    /// <param name="sizeColumns">Size of matrix at columns.</param>
+    /// <returns> Generated matrix.</returns>
+    public static int[,] GenerateMatrix(int sizeRows, int sizeColumns)
+    {
+        var matrix = new int[sizeRows, sizeColumns];
+
+        for (var i = 0; i < sizeRows; i++)
+        {
+            for (var j = 0; j < sizeColumns; j++)
+            {
+                matrix[i, j] = random.Next(1, 50);
+            }
+        }
+
+        return matrix;
+    }
+
+    /// <summary>
+    /// Get matrix.
+    /// </summary>
+    /// <returns>file matrix.</returns>
+    public int[,] GetMatrix()
+    {
+        return this.matrix;
+    }
```

Comment:
```suggestion
    public int[,] GetMatrix()
        => this.matrix;
```
И как раз это лучше было свойством сделать:
```csharp
    public int[,] Matrix
        => this.matrix;
```